x-pack/filebeat/docs/inputs/input-aws-s3.asciidoc

```diff
@@ -0,0 +1,95 @@
+[role="xpack"]
+
```

Comment:
There are some asciidoc coding problems here. Won't show up on CI because this topic isn't actually building. To get it building, you need to add a hook for the topic and fix a few issues. I've pushed the changes to my branch so you can grab the commit: https://github.com/elastic/beats/commit/a214e80915202e422cc41307cf7d4f98880bff5f. 

The content needs some editing and restructuring, but I didn't have time to do that because I had to fix the build errors first.

Reply:
@dedemorton Thanks for the commit!! That's why it shows red on my ide but ok with the build 😬